### ISSUE TITLE
Error in figRef: object 'result' not found

### ISSUE THREAD

**Issue by pat-s** [humburg/reportmd#26]
When referring to a child-document figure. What am I doing wrong here again? :/

Part4 yaml header: 

```yaml
output:
  reportMD::multi_document:
    toc: true
    fig_caption: true
    highlight: dracula
    depends:
      part2: vegetation-indices.Rmd
      part3: plot-speclibs.Rmd
    css: R.css
    use_namespace: false
```

The call is 

`r figRef('default-plot', target = 'part3')`

Part3 plot chunk: 

```r
{r default-plot, fig.cap="Default Plot", dependson='part2:speclib'}
plot(speclib)
```

full error:

```
Error in figRef("default-plot", target = "part3") : 
  object 'result' not found
Calls: <Anonymous> ... inline_exec -> withVisible -> eval -> eval -> figRef
In addition: Warning message:
In figRef("default-plot", target = "part3") :
  Unable to locate Figure with label 'default-plot' in file 'part3'.
Execution halted
```

Cross-references inside of each document are working.

**Comment by humburg** [comment]
At first glance that looks like it should be working. Could you provide the content of the file `plot-speclibs_figure.idx`?

**Comment by pat-s** [comment]
`plot-speclibs_figure.idx`
```
"default-plot"	"plot-speclibs"	"Figure 1"	"Default Plot"
"plot spectrum"	"plot-speclibs"	"Figure 2"	"10th spectrum of Speclib"
"multiple-speclibs"	"plot-speclibs"	"Figure 3"	"Multiple speclibs plots in one plot"
"combine-single-spectra"	"plot-speclibs"	"Figure 4"	"Multiple single spectra in one plot"
```

**Comment by humburg** [comment]
You didn't do anything wrong, this was a bug and should be fixed in version 0.3.2. Thanks for reporting it.
